### PR TITLE
feat: add diff to slack message and make wording more clear and consise

### PR DESCRIPTION
**Pull request by radiantspace**
As per discussion, we want to see the diff of what was deployed.

[Example message](https://github.slack.com/archives/C019LH97KJ9/p1598635429000500)
Screen Shot 2020-08-28 at 20 33 09

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
         environment_url: https://npmjs.com
         job_status: ${{job.status}}
         slack_token: ${{secrets.NPM_ROBOT_SLACK_TOKEN}}
-        slack_channel: npm-ops
+        slack_channel: npm-test-notifications
```
Comment (gordey4doronin) [inline .github/workflows/deploy.yml:23]: Need to revert that or not?

**Review by gordey4doronin — APPROVED** [review]
👍

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (radiantspace) [reply .github/workflows/deploy.yml:23]: i though maybe for this one we doesn't need that? like there are no real production deployments going on, or do you think its useful to have in npm-ops?

Comment (radiantspace) [reply .github/workflows/deploy.yml:23]: Lets keep that in npm-test-notifications for now, as this deployment is actually just a test.

Comment (gordey4doronin) [reply .github/workflows/deploy.yml:23]: 👍

**Pull request merged**